<ui version="4.0" >
 <class>KMinesThemeSelector</class>
 <widget class="QWidget" name="KMinesThemeSelector" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>521</width>
    <height>422</height>
   </rect>
  </property>
  <property name="maximumSize" >
   <size>
    <width>800</width>
    <height>600</height>
   </size>
  </property>
  <property name="windowTitle" >
   <string>Form</string>
  </property>
  <layout class="QGridLayout" >
   <item row="0" column="0" >
    <layout class="QHBoxLayout" >
     <item>
      <layout class="QVBoxLayout" >
       <item>
        <widget class="QListWidget" name="themeList" >
         <property name="minimumSize" >
          <size>
           <width>120</width>
           <height>0</height>
          </size>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="getNewButton" >
         <property name="text" >
          <string>&amp;Get New Themes</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <widget class="QGroupBox" name="groupBox_2" >
       <property name="title" >
        <string>Preview</string>
       </property>
       <layout class="QVBoxLayout" >
        <property name="spacing" >
         <number>6</number>
        </property>
        <property name="leftMargin" >
         <number>9</number>
        </property>
        <property name="topMargin" >
         <number>9</number>
        </property>
        <property name="rightMargin" >
         <number>9</number>
        </property>
        <property name="bottomMargin" >
         <number>9</number>
        </property>
        <item>
         <widget class="QLabel" name="themePreview" >
          <property name="sizePolicy" >
           <sizepolicy vsizetype="Preferred" hsizetype="Preferred" >
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="minimumSize" >
           <size>
            <width>160</width>
            <height>120</height>
           </size>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </item>
    </layout>
   </item>
   <item row="1" column="0" >
    <widget class="QGroupBox" name="groupBox" >
     <property name="title" >
      <string>Properties</string>
     </property>
     <layout class="QHBoxLayout" >
      <property name="spacing" >
       <number>6</number>
      </property>
      <property name="leftMargin" >
       <number>9</number>
      </property>
      <property name="topMargin" >
       <number>9</number>
      </property>
      <property name="rightMargin" >
       <number>9</number>
      </property>
      <property name="bottomMargin" >
       <number>9</number>
      </property>
      <item>
       <layout class="QVBoxLayout" >
        <property name="spacing" >
         <number>6</number>
        </property>
        <property name="leftMargin" >
         <number>0</number>
        </property>
        <property name="topMargin" >
         <number>0</number>
        </property>
        <property name="rightMargin" >
         <number>0</number>
        </property>
        <property name="bottomMargin" >
         <number>0</number>
        </property>
        <item>
         <widget class="QLabel" name="labelAuthor" >
          <property name="text" >
           <string>Author:</string>
          </property>
          <property name="alignment" >
           <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="labelContact" >
          <property name="text" >
           <string>Contact:</string>
          </property>
          <property name="alignment" >
           <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="labelDescription" >
          <property name="text" >
           <string>Description:</string>
          </property>
          <property name="alignment" >
           <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QVBoxLayout" >
        <property name="spacing" >
         <number>6</number>
        </property>
        <property name="leftMargin" >
         <number>0</number>
        </property>
        <property name="topMargin" >
         <number>0</number>
        </property>
        <property name="rightMargin" >
         <number>0</number>
        </property>
        <property name="bottomMargin" >
         <number>0</number>
        </property>
        <item>
         <widget class="QLabel" name="themeAuthor" >
          <property name="text" >
           <string/>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="themeContact" >
          <property name="text" >
           <string/>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="themeDescription" >
          <property name="text" >
           <string/>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <spacer>
        <property name="orientation" >
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" >
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </widget>
   </item>
   <item row="2" column="0" >
    <widget class="QLineEdit" name="kcfg_Theme" >
     <property name="enabled" >
      <bool>false</bool>
     </property>
     <property name="frame" >
      <bool>false</bool>
     </property>
     <property name="readOnly" >
      <bool>false</bool>
     </property>
    </widget>
   </item>
   <item row="3" column="0" >
    <spacer>
     <property name="orientation" >
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" >
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
